libs/rs-sdk-integration-tests/src/http.rs

```diff
@@ -63,3 +64,72 @@ pub fn test_http_post_success() {
 
     Process::error(&response.bytes);
 }
+
+/**
+ * Specify 1000ms timeout for an endpoint that responds in 5000ms.
+ */
+pub fn test_user_http_timeout() {
```

Comment:
Maybe we can make this `test_user_http_should_timeout`? To make it a little more clear it's the counterpart to the test below.